.github/workflows/yapf.yml

```diff
@@ -0,0 +1,34 @@
+name: Format with yapf
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'scripts/**.py'
```

Comment:
This will miss check for pull requests? I'm not sure if it will format the scripts submitted thorugh pr. Could you please figure it out?

If not, may be `on schedule` is a good choice instead.

Reply:
It makes sense and I will alter it.